src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.spec.ts

```diff
@@ -2,6 +2,7 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { SortableTableModule } from '../../sortable-table/sortable-table.module';
 import { McqQuestionStatisticsComponent } from './mcq-question-statistics.component';
+import { default as responses } from './test-data/mcqQuestionResponses.json';
```

Comment:
I think `import ResponseTestData from './test-data/mcqQuestionResponses.json'` would be better

Reply:
alright!